spec/html/proofer/fixtures/html/ignore_script_embeds.html

```diff
@@ -0,0 +1,3 @@
+<script type="text/html" id="navbar-logged-in" data-proofer-ignore>
```

Comment:
I would remove `data-proofer-ignore` from this fixture. The spec “ignores embeded scripts when asked” must pass without the attribute.